### PR TITLE
provider/aws: expose Route 53 zone nameservers

### PR DESCRIPTION
**Pull request by catsby**
:construction: 
This PR attempts to expose the NS elements of Route 53 zone. Refs #462

### CODE REVIEW

File: builtin/providers/aws/resource_aws_route53_zone.go [hunk 1]
```diff
@@ -87,6 +94,11 @@ func resourceAwsRoute53ZoneRead(d *schema.ResourceData, meta interface{}) error
 		return err
 	}
 
+	if rs.DelegationSet != nil {
+		d.Set("nameservers", rs.DelegationSet.NameServers)
+		log.Printf("\t@@@DEBUG: %#v:\t%#v\n", rs.DelegationSet.NameServers)
```
Comment (mitchellh) [inline builtin/providers/aws/resource_aws_route53_zone.go]: Debug line!

**Comment by mitchellh** [comment]
LGTM. It might be nice to add to the acceptance tests as well.

File: builtin/providers/aws/resource_aws_route53_zone.go (hunk 1 above)
Comment (catsby) [inline builtin/providers/aws/resource_aws_route53_zone.go]: :construction: :construction: :construction: :construction: :construction: :construction: :construction: :construction:

**Comment by catsby** [comment]
Example config for proof of concept:

```
provider "aws" {
   region = "us-west-2"
}

resource "aws_route53_zone" "main" {
	name = "notexample.com"
}

resource "aws_route53_record" "default" {
	zone_id = "${aws_route53_zone.main.zone_id}"
	name = "www.notexample.com"
	type = "A"
	ttl = "30"
	records = ["127.0.0.1"]
}

resource "aws_instance" "default" {
  ami = "ami-9fc29baf"
  instance_type = "t1.micro"
  tags {
    Name = "testing-user-data-things"
    TestData = "${aws_route53_zone.main.nameservers.1}"
  }
}
```

I believe this is blocked by #1215 though

**Comment by phinze** [comment]
This seems AOK - just needs an acceptance test line added to cover us for this attribute.

**Comment by catsby** [comment]
I'll revisit this when #1215 is resolved

**Comment by catsby** [comment]
close in favor of #1525

**Pull request closed without merging**